_includes/setup/android-setup-linux.md

```diff
@@ -0,0 +1,89 @@
+## Android setup
```

Comment:
Yes, we expect the Android setup instructions to change frequently. We have Q2 work planned to make this much simpler.